website/content/en/docs/deployment/cognito/guide-terraform.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Terraform Deployment Guide"
+description = "Deploy Kubeflow with AWS Cognito as identity provider using Terraform"
+weight = 30
++++
+
+> Note: Terraform deployment options are still in preview.
+
+## Prerequisites
+
+Be sure that you have satisfied the [installation prerequisites]({{< ref "../prerequisites.md" >}}) before working through this guide.
+
+Specifially, you must:
+- [Create a Ubuntu environment]({{< ref "../prerequisites/#create-ubuntu-environment" >}})
+- [Clone the repository]({{< ref "../prerequisites/#clone-repository" >}})
+- [Install the necessary tools]({{< ref "../prerequisites/#create-ubuntu-environment" >}})
+
+## Deployment Steps
```

Comment:
@rrrkharse will update this in a follow up PR once deployment options are finalized